src/design-system/asset-input/simple-asset-input.component.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import { Box } from '../box';
+import { Text } from '../text';
+
+import { AmountInput } from './amount-input.component';
+import { AmountInputAlignments } from './amount-input.css';
+import * as cx from './simple-asset-input.css';
+
+import type { Asset, AssetState } from './asset-input.data';
+
+interface Props {
+  state: AssetState;
```

Comment:
How about creating `SimpleAsset` type with a subset of properties that are used by the `SimpleAssetInput` so we don't have to provide data that is not used (e.g. `fiat`)